roles/nuage_master/templates/nuage-node-config-daemonset.j2

```diff
@@ -73,7 +76,7 @@ spec:
         # This container installs Nuage CNI binaries
         # and CNI network config file on each node.
         - name: install-nuage-cni
-          image: nuage/cni:{{ nuage_cni_container_image_version }}
+          image: {{ nuage_cni_image_version }}:{{ nuage_cni_container_image_version }}
```

Comment:
typo - should be `nuage_cni_image_name`?

Reply:
my bad.. you are right!! Thank you.